schema/swagger.yml

```diff
@@ -2353,6 +2353,9 @@ definitions:
       balance:
         type: string
         example: 1081560729258
+      username:
+        type: string
+        example: liskhq
```

Comment:
@robladbrook Username are not just a ordinary strings. We have a special string format `username` to match exact valid username; 

Look into swagger.yml file for its occurrences. e.g. 

https://github.com/LiskHQ/lisk/blob/2c0398a61bc350f1f9ec6acef29686c1edf370d3/schema/swagger.yml#L995-L1000

Remember to match this format the attribute must not be null. 

https://github.com/LiskHQ/lisk/blob/2c0398a61bc350f1f9ec6acef29686c1edf370d3/helpers/z_schema.js#L65-L71

So use below approach to set it to empty string if it appears to be null. 

https://github.com/LiskHQ/lisk/blob/2c0398a61bc350f1f9ec6acef29686c1edf370d3/api/controllers/voters.js#L134-L136